### ISSUE TITLE
Difficulty interpreting old-maps.co.uk file formats (WMS ?) 

### ISSUE THREAD

**Issue by JohnDeZoom** [lovasoa/dezoomify#146]
I've been trying to get a full size  image from the old-maps.co.uk site.
For example, this image:

[https://www.old-maps.co.uk/#/Map/357500/399500/13/101329](url)

the format of the links to tiles from Network monitoring is rather complex, as follows:-

[https://wms-1.old-maps.co.uk/api/wms?FORMAT=image%2Fjpeg&LAYERS=101329&TRANSPARENT=FALSE&SERVICE=WMS&VERSION=1.1.1&REQUEST=GetMap&STYLES=&SRS=EPSG%3A27700&BBOX=357500,399500,357600,399600&WIDTH=400&HEIGHT=400](url)

although other tiles have wms-2, or wms-3, or wms-4 prefixes !

So, is it possible to Dezoomify ? and if so what is the link that needs to be supplied ?

Thanks

John

**Comment by JohnDeZoom** [comment]
any ideas about this one please ?

**Comment by JohnDeZoom** [comment]
I assume the 'new site support' label means that it's been put on a wish list for treatment, not that it's necessarily done ?

**Comment by lovasoa** [comment]
Yes. You can have a look at the issues, a lot of people want new sites to be added.

If you want to add support for this site yourself, you can have a look at https://github.com/lovasoa/dezoomify/wiki/How-to-add-support-for-a-new-website

**Comment by AlexanderDeLargeBrotherSir** [comment]
@JohnDeZoom, try this again and if the issue is resolved then please very kindly close this issue.
